ci-scripts/build_nodejs_layer.sh

```diff
@@ -135,6 +135,13 @@ npm install -g copyfiles bestzip rimraf
 
 # Build layer
 pushd "./nodejs/packages/layer" > /dev/null
-npm run clean && npm install
+npm run clean && npm install --production
+# Dedupe dependencies to remove duplicates and reduce size
+npm dedupe
+# Remove unnecessary files to reduce layer size
+find node_modules -name "*.map" -delete
```

Comment:
That might affect stacktraces if they user is running with the appropriate flag, but I think it's a good call to remove the maps